protocols/yungapesquad/overview.md

```diff
@@ -0,0 +1,22 @@
+## Yung Ape DAO
+
+Yung Ape DAO is the governing body of the Yung Ape Squad that works in harmony with the project team and the community. The main responsibilities of the DAO are to track and approve expenditures, decide on project activations, fund growth on multiple verticals.
+
+## Yung Ape DAO Treasury
```

Comment:
I think that this heading and the one after should be grouped under `# Governance`

Reply:
fixed